js/LayersConfig.js

```diff
@@ -240,6 +240,61 @@ BR.LayersConfig = L.Class.extend({
         );
     },
 
+    renderOverpassPopupBody: function (overpassData) {
+        let output = '';
+
+        output += '<table class="overpass-tags">';
+
+        output += '<thead>';
+        output +=
+            '<tr><th class="overpass-label-key">' +
+            i18next.t('layers.overpass-table-key') +
+            '</th><th class="overpass-label-value">' +
+            i18next.t('layers.overpass-table-value') +
+            '</th></tr>';
+        output += '</thead>';
+
+        output += '<tbody>';
+        for (const key in overpassData.tags) {
+            if (key.substring(0, 5) === 'addr:') {
+                continue;
+            }
+            // `new Option().innerHTML` escapes HTML entities for XSS protection
+            let value = new Option(overpassData.tags[key]).innerHTML;
```

Comment:
Some afterthoughts on sanitizing, I can do a commit, but wanted to ask:
- we have [Util.sanitizeHTMLContent](https://github.com/nrenner/brouter-web/blob/f68d32b1f2b5ea34202c2bd62d7a91770942b7f0/js/Util.js#L137), is there a difference? Should we use that (current short enough to leave as is)?
- I would also sanitize the key, OSM API doesn't restrict allowed characters, see e.g. [TagInfo Characters in keys](https://taginfo.openstreetmap.org/reports/characters_in_keys#problem)